src/background-and-concepts.md

```diff
@@ -5,22 +5,22 @@ WebAssembly is a simple machine model and executable format with an [extensive
 specification].
 
 Although it has currently gathered attention in the JavaScript and web communities in general,
-it makes no assumptions about its host environment. Thus, it makes sense to think that _wasm_
+it makes no assumptions about its host environment. Thus, it makes sense to think that _Wasm_
```

Comment:
Unless you're referencing WebAssembly then it's lowercase as wasm! I had run into that as an issue myself when I wrote about it the first time.

Reply:
But we are referencing WebAssembly here?